cli/src/commands/merkle-tree/initialize.ts

```diff
@@ -0,0 +1,80 @@
+import { Args, Command } from "@oclif/core";
+import { PublicKey } from "@solana/web3.js";
+import {
+  CustomLoader,
+  getWalletConfig,
+  setAnchorProvider,
+} from "../../utils/utils";
+
+class InitializeCommand extends Command {
+  static description = "Initialize new Merkle Trees.";
+
+  static examples = ["light merkle-tree:initialize"];
+
+  static args = {
+    oldTransactionTree: Args.string({
```

Comment:
Couldn't we just pass the index of the new trees and generate the pubkeys automatically?
(The way it is right now this will be a pain to use, since you need to write a script to generate the new merkle tree pubkey first.)

Actually we could just fetch the current index of the latest Merkle tree from the chain and create the next higher one. Thus, we don't need any inputs at all and the command could be called initialize-next.

Reply:
I like the second idea - just fetching the current index and initializing the next Merkle tree.